CONTRIBUTING.md

```diff
@@ -335,11 +335,30 @@ Mocha tests are contained in `__tests__` directories.
 #### Jest
 Jest tests are stored in the same directory as source code files with the `.test.js` suffix.
 
-### Running Jest Unit Tests
-
-```bash
-node scripts/jest
-```
+### Running Kibana tests
+
+The following table outlines possible test file locations and how to invoke them:
+
+| Test location                                                                                                                                           | Runner command (working directory is kibana root)                                       |
```

Comment:
how about adding a column at the beginning of the table called “Type”? Values for each row could be “Jest”, “Mocha”, “Jest + API”, “Functional”, “Jest (X-Pack”), and “Integration (X-Pack)”

Reply:
I think having the rows be by type would help simplify this quite a bit. They would then correspond to the locations they cover and the command to execute.